src/plugins/nitroBypass.ts

```diff
@@ -88,11 +168,23 @@ export default definePlugin({
         return Boolean(UserStore.getCurrentUser().premiumType);
     },
 
+    getStickerLink(stickerId: string) {
+        return `${location.origin}/stickers/${stickerId}.png`;
```

Comment:
Discord has a map exported (just an object) with all the API endpoints as well as the asset endpoints.

You should probably look for that, just search through the sources for the discord assets link and you will find it.

Reply:
likely findByProps("STICKER")

Reply:
> likely findByProps("STICKER")

Returns this
![image](https://user-images.githubusercontent.com/24937357/199931389-cd91c0db-28da-4891-a72d-be26dc88c277.png)

Reply:
find(m => typeof m.STICKER === "function")

Reply:
You mean't like this? `STICKER` only returns `/stickers/INPUT` 🤔 

```suggestion
        const { STICKER } = find(m => typeof m.STICKER === "function");
        return `${location.origin}${STICKER(`${stickerId}.png`)}`;
```

Reply:
Well there's not really any issue with hardcoding it, it's not like it's gonna change

Reply:
You are getting the wrong webpack module, imma help when I'm on my pc